osu.Game/Graphics/ScreenshotManager.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Drawing.Imaging;
+using System.IO;
+using osu.Framework.Allocation;
+using osu.Framework.Configuration;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Input;
+using osu.Framework.Input.Bindings;
+using osu.Framework.Platform;
+using osu.Game.Configuration;
+using osu.Game.Input.Bindings;
+
+namespace osu.Game.Graphics
+{
+    public class ScreenshotManager : Container, IKeyBindingHandler<GlobalAction>, IHandleGlobalInput
+    {
+        private Bindable<ScreenshotFormat> screenshotFormat;
+        private GameHost host;
+        private Storage storage;
+
+        [BackgroundDependencyLoader]
+        private void load(GameHost host, OsuConfigManager config, Storage storage)
+        {
+            this.host = host;
+            this.storage = storage.GetStorageForDirectory(@"screenshots");
+
+            screenshotFormat = config.GetBindable<ScreenshotFormat>(OsuSetting.ScreenshotFormat);
+        }
+
+        public bool OnPressed(GlobalAction action)
+        {
+            switch (action)
+            {
+                case GlobalAction.TakeScreenshot:
+                    TakeScreenshot();

```

Comment:
There's no user feedback for this. Fire a notification?  Probably add the shutter sound from stable, too.

Reply:
I did not find shutter sample in the resources repo. Can you add it?